server/src/main/java/org/elasticsearch/ingest/IngestDocument.java

```diff
@@ -763,7 +751,7 @@ public static Object deepCopy(Object value) {
             for (Map.Entry<?, ?> entry : mapValue.entrySet()) {
                 copy.put(entry.getKey(), deepCopy(entry.getValue()));
             }
-            // TODO(stu): should this check for IngestSourceAndMetadata in addition to Map?
+            // TODO(stu): should this check for IngestCtxMap in addition to Map?
```

Comment:
Did you mean to leave this as part of the PR?

Reply:
It shows up because of the rename, it's still relevant question for now.